include/functions.h

```diff
@@ -975,29 +975,29 @@ void Health_InitData(GlobalContext* globalCtx);
+void Lights_PointSetInfo(LightInfo* info, s16 posX, s16 posY, s16 posZ, u8 red, u8 green, u8 blue,
                                 s16 radius, u32 type);
-void Lights_InitType0PositionalLight(LightInfoPositional* info, s16 posX, s16 posY, s16 posZ, u8 red, u8 green, u8 blue,
+void Lights_PointNoGlowSetInfo(LightInfo* info, s16 posX, s16 posY, s16 posZ, u8 red, u8 green, u8 blue,
                                      s16 radius);
-void Lights_InitType2PositionalLight(LightInfoPositional* info, s16 posX, s16 posY, s16 posZ, u8 red, u8 green, u8 blue,
+void Lights_PointGlowSetInfo(LightInfo* info, s16 posX, s16 posY, s16 posZ, u8 red, u8 green, u8 blue,
                                      s16 radius);
-void Lights_SetPositionalLightColorAndRadius(LightInfoPositional* info, u8 red, u8 green, u8 blue, s16 radius);
-void Lights_InitDirectional(LightInfoDirectional* info, s8 dirX, s8 dirY, s8 dirZ, u8 red, u8 green, u8 blue);
-void Lights_MapperInit(LightMapper* mapper, u8 red, u8 green, u8 blue);
-// ? func_8007A0B4(?);
-// ? func_8007A474(?);
-z_Light* Lights_FindFreeSlot();
-void func_8007A614(GlobalContext* globalCtx, LightingContext* lightCtx);
-void func_8007A698(LightingContext* lightCtx, u8 arg1, u8 arg2, u8 arg3, s16 arg4, s16 arg5);
-void Lights_SetAmbientColor(LightingContext* lightCtx, u8 red, u8 green, u8 blue);
-LightMapper* Lights_CreateMapper(LightingContext* lightCtx, GraphicsContext* gfxCtx);
-void Lights_ClearHead(GlobalContext* globalCtx, LightingContext* lightCtx);
-void Lights_RemoveAll(GlobalContext* globalCtx, LightingContext* lightCtx);
-z_Light* Lights_Insert(GlobalContext* globalCtx, LightingContext* lightCtx, void* info);
-void Lights_Remove(GlobalContext* globalCtx, LightingContext* lightCtx, z_Light* light);
-LightMapper* func_8007A960(GraphicsContext* gfxCtx, u8 red, u8 green, u8 blue);
-// ? func_8007A9B4(?);
-// ? func_8007ABBC(?);
+void Lights_PointSetColorAndRadius(LightInfo* info, u8 red, u8 green, u8 blue, s16 radius);
+void Lights_DirectionalSetInfo(LightInfo* info, s8 dirX, s8 dirY, s8 dirZ, u8 red, u8 green, u8 blue);
+void Lights_Reset(Lights* lights, u8 r, u8 g, u8 b);
+void Lights_Draw(Lights* lights, GraphicsContext* gfxCtxArg);
+LightNode* Lights_FindBufSlot();
+void Lights_InitContext(GlobalContext* globalCtx, LightContext* lightCtx);
+void func_8007A698(LightContext* lightCtx, u8 arg1, u8 arg2, u8 arg3, s16 arg4, s16 arg5);
+void Lights_SetAmbientColor(LightContext* lightCtx, u8 red, u8 green, u8 blue);
+Lights* Lights_New(LightContext* lightCtx, GraphicsContext* gfxCtx);
+void Lights_ClearHead(GlobalContext* globalCtx, LightContext* lightCtx);
+LightNode* Lights_Insert(GlobalContext* globalCtx, LightContext* lightCtx, LightInfo* info);
+void Lights_Remove(GlobalContext* globalCtx, LightContext* lightCtx, LightNode* light);
+Lights* Lights_AllocAndDraw(GraphicsContext* gfxCtx, u8 ambientR, u8 ambientG, u8 ambientB, u8 numLights, u8 r, u8 g, u8 b,
+                      s8 x, s8 y, s8 z);
```

Comment:
I think the rest of the parameters should line up with the `GraphicsContext*...` on the previous line